templates/OME-TIFF_Conversion.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    # Convert all the OME-TIFFs to FOVs
</pre><p>Can we change this comment to say "Convert all the OME-TIFFs to Single Channel TIFFs"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>